lib/Bindings/Python/circt/dialects/_hw_ops_ext.py

```diff
@@ -418,6 +418,31 @@ def create(elements):
     return hw.ArrayCreateOp(hw.ArrayType.get(type, len(vals)), vals)
 
 
+class ArrayConcatOp:
+
+  @staticmethod
+  def create(*args):
```

Comment:
args -> elems?

Reply:
I figured `*args` is standard python convention for variadic positional arguments, but I can give it a better name.